packages/react-components/react-tag-picker/bundle-size/Package.fixture.js

```diff
@@ -0,0 +1,7 @@
+import * as p from '@fluentui/react-tag-picker';
```

Comment:
based on offline convo @bsunderhus  - this was missing from the generator 

fixed https://github.com/microsoft/fluentui/pull/31325

Reply:
feel free to resolve @bsunderhus to unblock merging